Validation: Added msauthor and Table Headers (#226)
MicrosoftDocs/bingmaps-docs: BingMaps/styling/map-style-sheet-entry-properties.md

---
title: Map Style Sheet Entry Properties | Microsoft Docs

> description: The properties listed below can be set on entries in a map style sheet to customize the appearance of a map in various Microsoft map controls.

Reviewer: ```suggestion
description: Learn about the map style sheet entry properties that can be set on entries in a map style sheet to customize the maps appearance in various Microsoft map controls.
```
